mlflow/openai/_agent_tracer.py

```diff
@@ -73,79 +79,50 @@ def __init__(
     ) -> None:
         super().__init__(**kwargs)
         self._span_id_to_mlflow_span: dict[str, LiveSpan] = {}
```

Comment:
```suggestion
        self._span_id_to_mlflow_span: dict[str, SpanWithToken] = {}
```

can we use: 

https://github.com/mlflow/mlflow/blob/811c68ea23b63e51a30208817ea5f87dfd5861bc/mlflow/tracing/utils/token.py#L8

?